tests/integration/hpe_3par_api_file_persona_test.py

```diff
@@ -0,0 +1,321 @@
+
+PLUGIN_TYPE = cfg['plugin']['type']
+HOST_OS = cfg['platform']['os']
+THIN_SIZE = cfg['volumes']['thin_size']
+FULL_SIZE = cfg['volumes']['full_size']
+DEDUP_SIZE = cfg['volumes']['dedup_size']
+COMPRESS_SIZE = cfg['volumes']['compress_size']
+
+if PLUGIN_TYPE == 'managed':
+    HPE3PAR = cfg['plugin']['managed_plugin_latest']
+    CERTS_SOURCE = cfg['plugin']['certs_source']
+else:
+    HPE3PAR = cfg['plugin']['containerized_plugin']
+    PLUGIN_IMAGE = cfg['plugin']['containerized_image']
+    if HOST_OS == 'ubuntu':
+        PLUGIN_VOLUMES = cfg['ubuntu_volumes']
+    elif HOST_OS == 'suse':
+        PLUGIN_VOLUMES = cfg['suse_volumes']
+    else:
+        PLUGIN_VOLUMES = cfg['rhel_volumes']
+
+@requires_api_version('1.21')
+class FilePersonaTest(HPE3ParBackendVerification,HPE3ParVolumePluginTest):
+
+    @classmethod
+    def setUpClass(cls):
+        if PLUGIN_TYPE == 'managed':
+            c = docker.APIClient(
+                version=TEST_API_VERSION, timeout=600,
+                **docker.utils.kwargs_from_env()
+                )
+            try:
+                prv = c.plugin_privileges(HPE3PAR)
+                logs = [d for d in c.pull_plugin(HPE3PAR, prv)]
+                assert filter(lambda x: x['status'] == 'Download complete', logs)
+                if HOST_OS == 'ubuntu':
+                    c.configure_plugin(HPE3PAR, {
+                        'certs.source': CERTS_SOURCE
+                    })
+                else:
+                    c.configure_plugin(HPE3PAR, {
+                        'certs.source': CERTS_SOURCE,
+                        'glibc_libs.source': '/lib64'
+                    })
+                pl_data = c.inspect_plugin(HPE3PAR)
+                assert pl_data['Enabled'] is False
+                while pl_data['Enabled'] is False:
+                    c.enable_plugin(HPE3PAR)
+                    HPE3ParBackendVerification.hpe_wait_for_all_backends_to_initialize(cls, driver=HPE3PAR, help='backends')
+                pl_data = c.inspect_plugin(HPE3PAR)
+                assert pl_data['Enabled'] is True
+            except docker.errors.APIError:
+                pass
+        else:
+            c = docker.from_env(version=TEST_API_VERSION, timeout=600)
+            try:
+                mount = docker.types.Mount(type='bind', source='/opt/hpe/data',
+                                           target='/opt/hpe/data', propagation='rshared'
+                )
+                c.containers.run(PLUGIN_IMAGE, detach=True,
+                                 name='hpe_legacy_plugin', privileged=True, network_mode='host',
+                                 restart_policy={'Name': 'on-failure', 'MaximumRetryCount': 5},
+                                 volumes=PLUGIN_VOLUMES, mounts=[mount],
+                                 labels={'type': 'plugin'}
+                )
+                HPE3ParBackendVerification.hpe_wait_for_all_backends_to_initialize(cls, driver=HPE3PAR, help='backends')
```

Comment:
Please check if the utility method will query all the backends -- both block + file backends .. i believe it's only block backends status is queried..